modules/virt-cloning-a-datavolume.adoc

```diff
@@ -33,29 +30,25 @@ spec:
   source:
     pvc:
       namespace: "<source-namespace>" <2>
-      name: "<my-favorite-vm-disk>" <3>
-  pvc:
-    accessModes:
-      - ReadWriteMany
+      name: "<my-favorite-vm-disk>" <3>)
+  storage: <4>
     resources:
       requests:
-        storage: <2Gi> <4>
-    volumeMode: Block <5>
+        storage: <2Gi> <5>
 ----
 <1> The name of the new data volume.
 <2> The namespace where the source PVC exists.
 <3> The name of the source PVC.
-<4> The size of the new data volume. You must allocate enough space, or the
-cloning operation fails. The size must be the same as or larger than the source PVC.
-<5> Specifies that the destination is a block PV.
+<4> Specifies allocation with the *storage* API
+<5> The size of the new data volume.
 
 . Start cloning the PVC by creating the data volume:
 +
 [source,terminal]
 ----
 $ oc create -f <cloner-datavolume>.yaml
 ----
-+
+
 [NOTE]
```

Comment:
Just double-checking that it's intentional that you want the `[NOTE]` to appear "outside" of the ordered list / procedure, instead of attached to the 2nd step.

Reply:
Added a +